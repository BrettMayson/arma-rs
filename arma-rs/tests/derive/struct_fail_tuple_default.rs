use arma_rs_proc::FromArma;

#[derive(FromArma)]
#[arma(default)]
pub struct DeriveTest(String, u32);

pub fn main() {}
