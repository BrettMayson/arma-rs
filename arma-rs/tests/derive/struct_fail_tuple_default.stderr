error: #[arma(default)] can only be used on structs with named fields
 --> tests/derive/struct_fail_tuple_default.rs:3:10
  |
3 | #[derive(FromArma)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `FromArma` (in Nightly builds, run with -Z macro-backtrace for more info)

error: #[arma(default)] can only be used on structs with named fields
 --> tests/derive/struct_fail_tuple_default.rs:7:10
  |
7 | #[derive(FromArma)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `FromArma` (in Nightly builds, run with -Z macro-backtrace for more info)
